docs-2.0/5.configurations-and-logs/1.configurations/3.graph-config.md

```diff
@@ -92,3 +92,15 @@ For all parameters and their current values, see [Configurations](1.configuratio
 | ------------------- | ---------------- | ------------------------------------------ |
 |`enable_authorize`   |`false`           |When set to `false`, the system authentication is not enabled. For more information, see [Authentication](../../7.data-security/1.authentication/1.authentication.md).|
 |`auth_type`          |`password`        |Specifies the login method. Available values are `password`, `ldap`, and `cloud`.|
+
+## Experimental configurations
+
+| Name                | Predefined value         | Description                                |
+| ------------------- | ------------------------ | ------------------------------------------ |
+|`enable_experimental_feature`|`false`| Specifies the experimental feature. Optional values are `true` and `false`. For currently supported experimental features, see below.|
+
+### Experimental features
+
+| Name                 |  Description                             |
+| ------------------- | ------------------------------------------|
+| TOSS | The TOSS (Transaction on Storage Side) function is used to ensure the final consistency of the `INSERT`, `UPDATE`, or `UPSERT` operations on edges (because one edge logically corresponds to two key-value pairs on the hard disk). The `DELETE` operation is not currently supported. After the TOSS function is enabled, the time delay of related operations will increase by about one time. |
```

Comment:
is it better to use the passive sentence here? can it be changed from will increase to will be increased?